Mission-Feasibility-Study/SPEXMissionFeasibilityStudy.tex

```diff
@@ -98,11 +93,9 @@
 \hyphenation{explor-ation}
 
 \begin{abstract}
-  A standard format for Project Design Documents is key to organize and document the many projects members of RIT Space Exploration wish to pursue.
-  The goal of the SPEX Standard is to organize, refine, and archive space exploration research.
```

Comment:
I know it's trivial, but acronyms should be spelled out the first time they are used with the acronym in parentheses. `RIT Space Exploration (SPEX)`

Alternatively, place commonly used acronyms in a "Nomenclature" section and define them there